improver_tests/metadata/test_utilities.py

```diff
@@ -471,5 +474,66 @@ def test_errors_get_model_id_attr(cubes: List[Cube], method: Callable, message):
         get_model_id_attr(cubes, model_id_attr)
 
 
+@pytest.fixture
+def data_times():
+    """Define the times for the input cubes. The bounds are set to be
+    non-conformant with the IMPROVER standards, such that the point
+    is the start of the period rather than the end."""
+    frt = datetime(2025, 1, 15, 3, 0)
+    times = []
+    for hour in range(3, 9 + 1, 3):
+        time = frt + timedelta(hours=hour)
+        bounds = [time, time + timedelta(hours=3)]
+        times.append((frt, time, bounds))
+    return times
+
+
+@pytest.fixture
+def multi_time_cube(data_times):
+    """Create a cube that has a time coordinate with an entry for
+    each validity time passed in. Adds bounds to the
+    forecast_reference_time purely to demonstrate the functionality."""
+
+    data = 281 * np.ones((3, 3)).astype(np.float32)
+    cubes = CubeList()
+    for frt, time, time_bounds in data_times:
+        cubes.append(
+            set_up_variable_cube(data, time=time, time_bounds=time_bounds, frt=frt)
+        )
+    cube = cubes.merge_cube()
+    frt_crd = cube.coord("forecast_reference_time")
+    frt_crd.bounds = [[frt, frt + 3600] for frt in frt_crd.points]
+    return cube
+
+
+def test_enforce_time_point_standard(multi_time_cube, data_times):
```

Comment:
I think that the only test that you could add would be something to explicitly check that the code works when one of the coordinates in the `TIME_COORDS` dictionary isn't present. This is actually being tested in both of these tests, as `blend_time` isn't present in the cube that you've created for testing anyway, but it might be clearer to make this explicit.